pkg/network/netlink/conntracker.go

```diff
@@ -108,14 +116,17 @@ func NewConntracker(config *config.Config) (Conntracker, error) {
 
 func newStats() stats {
 	return stats{
-		gets:                 atomic.NewInt64(0),
-		getTimeTotal:         atomic.NewInt64(0),
-		registers:            atomic.NewInt64(0),
-		registersDropped:     atomic.NewInt64(0),
-		registersTotalTime:   atomic.NewInt64(0),
-		unregisters:          atomic.NewInt64(0),
-		unregistersTotalTime: atomic.NewInt64(0),
-		evicts:               atomic.NewInt64(0),
+		gets:                     nettelemetry.NewStatGaugeWrapper("conntracker", "gets", []string{}, "description"),
+		getTimeTotal:             atomic.NewInt64(0),
+		registers:                nettelemetry.NewStatGaugeWrapper("conntracker", "registers", []string{}, "description"),
+		registersDropped:         atomic.NewInt64(0),
+		registersTotalTime:       atomic.NewInt64(0),
+		unregisters:              atomic.NewInt64(0),
+		unregistersTotalTime:     atomic.NewInt64(0),
+		evicts:                   nettelemetry.NewStatGaugeWrapper("conntracker", "evicts_total", []string{}, "description"),
+		nanoSecondsPerGet:        telemetry.NewGauge("conntracker", "nanoSecondsPerGet", []string{}, "description"),
```

Comment:
You want to use snake case for names you pass to `NewGauge`.

Reply:
I believe this was done everywhere